setup.py

```diff
@@ -41,7 +41,8 @@ def get_package_metadata(attribute):
     },
     install_requires=[
         'requests >= 2.10',
-        'requests-jwt >= 0.4, < 0.5'
+        'requests-jwt >= 0.4, < 0.5',
+        'pyjwt == 1.7.1'
```

Comment:
Python 3.7 should be added to the package metadata

Reply:
This PR does not add tests for python 3.7 (I just bumped the `docs` tox environment to 3.7), because they are failing. We can add that in a separate patch, I just wanted to fix tests for currently supported versions

Reply:
> https://github.com/tgs/requests-jwt

>This package is not actively maintained and has never had a security audit.

Reply:
> This PR does not add tests for python 3.7

Sorry, I read the diff too fast and thought the Travis environment matrix was updated to run the CI on Python 3.7 :eyes: 

> This package is not actively maintained and has never had a security audit

Maybe worth raising a dedicated issue to find a suitable replacement :+1: